modin/pandas/test/dataframe/test_map_metadata.py

```diff
@@ -921,19 +923,7 @@ def test_dropna_subset_error(data, axis, subset):
 
 
 @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
-@pytest.mark.parametrize(
-    "astype",
-    [
-        "category",
-        pytest.param(
-            "int32",
-            marks=pytest.mark.xfail(
-                reason="Modin astype() does not raises ValueError at non-numeric argument when Pandas does."
```

Comment:
This is fixed?

Reply:
it seems, I've removed `xfail` decorator and it passed)